.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+* @bcumming @msimberg
+docs/software/sciapps/cp2k @abussy @RMeli
```

```suggestion
* @bcumming @msimberg @RMeli
/docs/software/sciapps @RMeli
/docs/software/sciapps/cp2k @abussy
```
> However, I think for the top level * code owners it'd be good to have a few more people.

I'm watching all activity on the repository anyways, so if you want more peoples on the top level I'm fine with it.

---

Thank you! https://github.com/eth-cscs/cscs-docs/pull/45/commits/940495f45133e52f7df083ff38921a92ad47c83a